docs/en_US/NAS/QuickStart.rst

```diff
@@ -52,6 +52,8 @@ Below is a very simple example of defining a base model.
       output = F.log_softmax(x, dim=1)
       return output
 
+.. tip:: Always keep in mind that you should use ``import nni.retiarii.nn.pytorch as nn`` and ``@model_wrapper``. Many mistakes are a result of forgetting one of those. Also, please use ``torch.nn`` for submodules of ``nn.init``, e.g., ``torch.nn.init`` instead of ``nn.init``. 
```

Comment:
what is the difference between `@module_wrapper` and `@nni.trace`?

Reply:
I mentioned nothing about `nni.trace` here.

`@model_wrapper` used `@nni.trace` as its underlying implementation, but they are totally different APIs.
`@model_wrapper` marks a module as a search space (which will reset label counting and do many other stuffs). `@nni.trace` is to record parameters and that's all.